src/Compilers/VisualBasic/Test/Emit/PDB/PDBExternalSourceDirectiveTests.vb

```diff
@@ -69,8 +69,8 @@ End Class
         </method>
```

Comment:
So this is the case where we hit the all hidden sequence points case. Seems like it will actually be rather common in VB case. I'd avoid emitting any sequence points in that case.